### PR TITLE
v1.13.0. Dependent validation. Clear code.

### PR DESCRIPTION
**Pull request by shystruk**

### CODE REVIEW

File: src/csv-file-validator.js [hunk 1]
```diff
@@ -79,9 +78,11 @@
 						if (valueConfig.name !== columnValue) {
 							file.inValidMessages.push(
 								_isFunction(valueConfig.headerError)
-									? valueConfig.headerError(columnValue, valueConfig.name, rowIndex + 1, columnIndex + 1)
-									: 'Header name ' + columnValue + ' is not correct or missing in the ' + (rowIndex + 1) + ' row / '
-										+ (columnIndex + 1) + ' column. The Header name should be ' + valueConfig.name
+									? valueConfig.headerError(
+										columnValue, valueConfig.name, rowIndex + 1, columnIndex + 1
+									)
+									: `Header name ${columnValue} is not correct or missing in the ${rowIndex + 1} row/
```
Comment (camerondunford) [inline src/csv-file-validator.js:84]: @shystruk Are the whitespace changes here (before/after the slash) necessary changes? They're breaking some of my tests. 😅

Comment (shystruk) [reply src/csv-file-validator.js:84]: @camerondunford small updates on the error messages. Hope you don't have a lot of test cases to update ;)

Comment (camerondunford) [reply src/csv-file-validator.js:84]: Thanks. It seems like an inconsistent (before/after the slash) & excessive (after the slash) application of whitespace.


Default (-zsh) 2021-12-07 15-40-47

Comment (shystruk) [reply src/csv-file-validator.js:84]: oh, looks not right. Will fix that. Thanks!

Comment (shystruk) [reply src/csv-file-validator.js:84]: @camerondunford go and upgrade to 1.13.1

Comment (camerondunford) [reply src/csv-file-validator.js:84]: Right on, thank you! 🙇

**Pull request merged**